scripts/qa_crowbarsetup.sh

```diff
@@ -1740,8 +1740,24 @@ function install_crowbar_init()
     wait_for 100 3 "onadmin_is_crowbar_init_api_available" "crowbar init service to start"
 }
 
+function onadmin_finalize_server_upgrade()
+{
+    if iscloudver 5minus; then
+        complain 11 "This upgrade path is only supported for Cloud 6+"
+    fi
+
+    # make sure that crowbar schemas are up to date
+    pushd /opt/dell/crowbar_framework
+    sudo -u crowbar RAILS_ENV=production bin/rake crowbar:schema_migrate_prod
```

Comment:
It's true it won't be needed only by mkcloud. Can't you make it mart of db migrate call?